### PR TITLE
Create ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by luja-c8y

Ticket: [MTM-58987](https://cumulocity.atlassian.net/browse/MTM-58987)
Original PR: [7745](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7745)

[MTM-58987]: https://cumulocity.atlassian.net/browse/MTM-58987?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: As a developer i want to migrate data points table widget to the angular (#6391) [GRAFT][release/cd] (#7745)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md]: The Claude AI suggested to use the following title:
```suggestion
title: Migrated Data points table widget to Angular
```

File: content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58987
+version: 1021.21.0
+---
+As a developer i want to migrate data points table widget to the angular (#6391) [GRAFT][release/cd] (#7745)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md]: The Claude AI suggested to use the following body:
```suggestion
As part of the ongoing modernization efforts to migrate the UI to Angular, the data points table widget has now been migrated from the old technology to Angular. This change improves the performance, maintainability and consistency of the widget with the rest of the application. The functionality and behavior of the data points table widget remains the same, so there should be no impact on end users.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md (hunk 2 above)
Comment (luja-c8y) [reply content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md]: Instead `old technology` I would write `AngularJS`, but it's a minor improvements of claude's suggestion:
```suggestion
As part of the ongoing modernization efforts to migrate the UI to Angular, the data points table widget has now been migrated from the AngularJS to Angular. This change improves the performance, maintainability and consistency of the widget with the rest of the application. The functionality and behavior of the data points table widget remains the same, so there should be no impact on end users.
```

**Review by luja-c8y — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58987
+version: 1021.21.0
+---
+As part of the ongoing modernization efforts to migrate the UI to Angular, the data points table widget has now been migrated from the AngularJS to Angular. This change improves the performance, maintainability and consistency of the widget with the rest of the application. The functionality and behavior of the data points table widget remains the same, so there should be no impact on end users.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md]:
```suggestion
As part of the ongoing modernization efforts to migrate the UI to Angular, the "Data points table" widget has now been migrated from AngularJS to Angular. This change improves the performance, maintainability and consistency of the widget with the rest of the application. The functionality and behavior of the "Data points table" widget remains the same, so there is no impact on users.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@rahulc8y or @janhommes Please approve

File: content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58987
+version: 1021.21.0
+---
+As part of the ongoing modernization efforts to migrate the UI to Angular, the "Data points table" widget has now been migrated from AngularJS to Angular. This change improves the performance, maintainability and consistency of the widget with the rest of the application. The functionality and behavior of the "Data points table" widget remains the same, so there is no impact on users.
```
Comment (rahulc8y) [inline content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md]:
```suggestion
As part of the ongoing modernization efforts to migrate the UI to Angular, the "Data points table" widget has now been migrated from AngularJS to Angular. This change improves the performance, maintainability and consistency of the widget with the rest of the application. The widget now offers enhanced features including:

- Configurable decimal place precision for data point visualization
- Data export capabilities to CSV/XLSX formats
- Improved responsiveness across different screen sizes and devices
- Auto-refresh based on configurable interval

All existing functionality remains unchanged, ensuring a seamless transition for current users.
```

File: content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Migrated Data points table widget to Angular
```
Comment (rahulc8y) [inline content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md]:
```suggestion
title: Enhanced Data points table widget
```

File: content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58987
+version: 1021.21.0
+---
+As part of the ongoing modernization efforts to migrate the UI to Angular, the "Data points table" widget has now been migrated from AngularJS to Angular. This change improves the performance, maintainability and consistency of the widget with the rest of the application. The widget now offers enhanced features including:
+
+- Configurable decimal place precision for data point visualization
+- Data export capabilities to CSV/XLSX formats
```
Comment (luja-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-21-0-as-a-developer-i-want-to-migrate-data-points-table-widget.md]:
```suggestion
```
Integration is not yet merged: https://cumulocity.atlassian.net/browse/MTM-59872

**Pull request merged**